### PR TITLE
Consolidate xdebug image into dev

### PR DESCRIPTION
**Pull request by nickschuch**

### CODE REVIEW

**Comment by kimpepper** [comment]
Do we need nginx changes too?

File: dev/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+ARG PHP_VERSION=8.0
 
 ARG ARCH=amd64
 
 USER root
 
+RUN apk add php${PHP_VERSION}-xdebug
+
 # https://blog.blackfire.io/alpine-linux-support.html
 RUN PHP_VERSION=$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;") && \
```
Comment (kimpepper) [inline dev/Dockerfile:12]: No need for looking up php version if its now an arg.

Comment (nickschuch) [reply dev/Dockerfile:12]: The formatting is a little different between the 2:

ARG = 8.1
php -r approach = 81

Will strip the `.` using bash.

**Review by kimpepper — CHANGES_REQUESTED** [review]
No need for looking up php version if its now an arg.

**Review by kimpepper — APPROVED** [review]

**Pull request merged**